src/custom-element.js

```diff
@@ -1679,22 +1681,14 @@ function createBaseCustomElementClass(win) {
     /**
      * Turns the loading indicator on or off.
      * @param {boolean} state
-     * @param {{cleanup:(boolean|undefined), force:(boolean|undefined), startTime:(number|undefined)}=} opt_options
+     * @param {{cleanup:(boolean|undefined), startTime:(number|undefined)}=} opt_options
      * @public @final
      */
     toggleLoading(state, opt_options) {
       const cleanup = opt_options && opt_options.cleanup;
-      const force = opt_options && opt_options.force;
       const startTime = opt_options && opt_options.startTime;
       assertNotTemplate(this);
-      if (
```

Comment:
I think this check is done here to account for async wait times. I'm not sure we can remove this.

Reply:
What async wait times? Can you explain more? 
Also, do you know what the `CommonSignals.RENDER_START` part is for?

Why shouldn't this case be handled by `this.isLoadingEnabled_()`

Reply:
I think Justin means if the element enters viewport (start 100ms timer), and then renders itself within 100ms. We won't want to display the loading indicator since the element has already fully rendered.

Reply:
I'm probably missing something, but wouldn't that case be caught (more correctly too) by https://github.com/ampproject/amphtml/pull/27508/files/5b7d50338956b25003f4b6ac2f4b2c0b6026d226#diff-8e1cdba946c07a7e631ef74645a5d6f0R1702

Reply:
Ya sorry I was just answering the first part to your question.

Reply:
I was confused, ignore me.